tools/rpi-sign-bootcode

```diff
@@ -105,6 +108,28 @@ class ImageFile:
         debug("%08x %20s: [%6d] %s" % (self.pos(), 'RSA', len(arr), pem_file))
         self.append(arr)
 
+        h = SHA256.new()
+        h.update(key.n.to_bytes(256, byteorder='little'))
+        h.update(key.e.to_bytes(8, byteorder='little'))
```

Comment:
Looks like you could maybe just do `h.update(arr)` here?

Reply:
I think it's clearer this way